examples/ct/serialize_test.go

```diff
@@ -68,7 +60,7 @@ func TestSignV1SCTForCertificate(t *testing.T) {
 	}
 
 	if !reflect.DeepEqual(got, expected) {
-		t.Fatalf("Mismatched SCT (cert), got %v, expected %v", got, expected)
+		t.Fatalf("Mismatched SCT (cert), got \n%v, expected \n%v", got, expected)
```

Comment:
Using something like https://github.com/d4l3k/messagediff would give the nicest failure messages, but I don't expect you to add that in this PR.